test/helpers/slow-data-stream.ts

```diff
@@ -0,0 +1,18 @@
+import {PassThrough} from 'stream';
+
+export default () => {
```

Comment:
It'll be nice to add an explicit return type here so it's once less thing we have to add when making the typescript types better.
```suggestion
export default (): PassThrough => {
```